recipes/bxdecay0/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -o xtrace -o nounset -o pipefail -o errexit
+
+if [[ "${DEBUG_C:-no}" == "yes" ]]; then
+  CMAKE_BUILD_TYPE=Debug
+else
+  CMAKE_BUILD_TYPE=Release
+fi
```

Comment:
Can you explain who is supposed to set this `DEBUG_C` env variable? I am not sreu if this would be expected by most mantainers of conda-forge, perhaps it is better to omit it?

Reply:
This was probably copied from the geant4-feedstock, not sure where that came from.